docs/docs/tutorials/debugging.md

```diff
@@ -0,0 +1,89 @@
+# :material-bug: Hardware Debugging
+
+## Background
+
+For debugging our boards, we use OpenOCD and GDB.
+
+[OpenOCD (Open On-Chip Debugger)](https://openocd.org/) is a tool that enables on-chip debugging, flash programming, and testing for microcontrollers. It will start a server that listens for commands from various sources such as GDB.
+
+[GDB (GNU Debugger)](https://www.sourceware.org/gdb/) is a debugging tool that enables inspection, modification, and control of program execution on microcontrollers.
+
+## Installation
+
+Use the following command to install OpenOCD:
+
+=== "Windows"
+    ```text
+    <!-- TODO: Rida documentation for OpenOCD -->
+    ```
+
+=== "Linux / Mac"
+    ```text
+    brew install openocd
```

Comment:
brew isn't standard on Linux. the [openocd docs](https://openocd.org/pages/getting-openocd.html) suggest using `sudo apt install openocd`

Reply:
I don't need you to verify that this command works on linux.